doc/development_process/project_roles.rst

```diff
@@ -49,6 +48,26 @@ Contributors are granted the following rights and responsibilities:
 * Responsibility to follow the project’s code of conduct
   (https://github.com/zephyrproject-rtos/zephyr/blob/main/CODE_OF_CONDUCT.md)
 
+Contributors are initially only given `Read
+<https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization>`_
+access to the Zephyr GitHub repository. Specifically, at the Read access level,
+Contributors are not allowed to assign reviewers to their own pull requests. An
+automated process will assign reviewers, or you may share the pull request on
+the `Zephyr devel mailing list <https://lists.zephyrproject.org/g/devel>`_.
```

Comment:
Another thing people do is ask for reviews on Discord, if you want to add a link to that here as well. Not a blocking comment, just something that occurred to me.